src/grandorgue/dialogs/GOMidiListDialog.cpp

```diff
@@ -29,7 +29,7 @@ GOMidiListDialog::GOMidiListDialog(
   wxWindow *parent,
   const std::vector<GOMidiConfigurator *> &midi_elements)
   : GOSimpleDialog(
-    parent,
+    parent, wxT("MIDI Objects"), _("MIDI Objects"), 0, wxOK | wxHELP),
     wxT("MIDI Objects"),
```

Comment:
But you have to remove the next lines, otherwise a compilation error will occure.

Reply:
Sorry, I overlooked this.